test/e2e/app-dir/actions/app-action-progressive-enhancement.test.ts

```diff
@@ -35,7 +35,7 @@ createNextDescribe(
 
       await check(() => {
         return browser.eval('window.location.pathname + window.location.search')
-      }, '/header?name=test&constructor=_FormData&hidden-info=hi')
```

Comment:
We already have `formData.get('name')` covered, there's no need to test the constructor function's name as that might differ in different Node.js versions and polyfills.